docs/protocol/extension_statistics.md

```diff
@@ -111,8 +112,16 @@ $model_stat =
   DOES NOT include cache hits.
 
 - "inference_stats" : The aggregate statistics for the
-  model/version. So, for example, "inference_stats":"success"
-  indicates the number of successful inference requests for the model.
+  model. So, for example, "inference_stats":"success" indicates the number of
+  successful inference requests for the model.
+
+- "response_stats" : The aggregate decoupled response statistics for the
+  model. For example, { "key" : { "response_stats" : "success" } } indicates the
+  aggregate statistics of successful decoupled responses at "key" for the model,
+  where "key" identifies each decoupled response generated by the model across
+  different requests. For example, given a model that generates three responses,
+  the key could be "0", "1" and "2" identifying the three responses in order.
+  Check with the model backend on the keys it can return and their meanings.
```

Comment:
The "keys" aren't backend specific are they? Isn't it a fixed set defined in core/proto?

Reply:
They are not backend specific, at this point, unless we allow customizable keys for the backend in the future. I only left the example behind, I think that should be enough.